target/linux/tegra/base-files/lib/tegra.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
> Please make it mandatory. Add ``REQUIRE_IMAGE_METADATA=1`` to the top of ``target/linux/tegra/base-files/lib/upgrade/platform.sh``.

Done.

> The image metadata boardname is validated against /tmp/sysinfo/board_name, which is the compatible string from the dts. If you want to support multiple boards with one image you need to add all of them (space separated) to the ``SUPPORTED_DEVICES``.
> 
> Either I get something wrong or it never worked as intended.

Yes that is right, but seems it also validates contents of ``/tmp/sysinfo/model`` (in my case ``Compulab TrimSlice board``) with string ``"board": "tegra"`` in metadata, and thus this fails. The sunxi images have similar structure, maybe someone can test it invoking ``sysupgrade -v <lede-sunxi-image>``.